start/resources/singularity-containers.md

```diff
@@ -88,170 +27,153 @@ tools and libraries. The images are are:
+To get your images included, please create a git pull request with the container 
+identifier in `docker_images.txt` in the
+[cvmfs-singularity-sync repository](https://github.com/opensciencegrid/cvmfs-singularity-sync), 
+or contact
+[support@osgconnect.net](mailto:support@osgconnect.net)
+and we can help you.
 
-    $ docker push namespace/repository_name
+Once your submission has been accepted, it will be automatically converted to a Singularity 
+image and pushed to the CVMFS Singularity repository.  Note: some 
+common Dockerfile features, like ENV and ENTRYPOINT, are ignored when the Docker 
+image is converted to a Singularity image. See our [how to guide]() for more details 
+of how to deal with this. 
 
-#### Editing the default image using local Docker
+Once your container has been added to CVMFS, 
+if you update your original Docker image, new versions pushed to Docker Hub will
+automatically be detected and the version on the OSG (in the CVMFS filesystem)
+will be updated accordingly.
 
-To create an image locally using Docker, you will need to have Docker 
-installed on your machine. For install instructions, see the 
-[Docker documentation](https://docs.docker.com/engine/installation/#server).
+### Using Your Container
 
-With docker installed you can now get the desired image from Docker Hub. Again, we will
-look at OSG Ubuntu Xenial image. 
+To use your container to run jobs, you will follow the same steps as above (under "Using OSG 
+Provided Singularity Images"), but will change the `+SingularityImage` option to 
+include your container identifier, like so: 
 
-    $ docker pull opensciencegrid/osgvo-ubuntu-xenial
+    +SingularityImage = "/cvmfs/singularity.opensciencegrid.org/namespace/repository_name"
 
-We will run the image in a docker interactive session
+For example, if my Docker Hub username was `alice` and I created a container called 
+`ncbi-blast`, my submit file might look like this: 
 
-    $ docker run -it --name <docker_session_name_here> opensciencegrid/osgvo-ubuntu-xenial /bin/bash
+    universe = vanilla
+    executable = job.sh
+	
+    Requirements = HAS_SINGULARITY == TRUE
+    +SingularityImage = "/cvmfs/singularity.opensciencegrid.org/alice/ncbi-blast"
 
-Giving the session a name is important because it will make it easier to 
-reattach the session later and commit the changes later on. Now you will 
-be greeted by a new command line prompt that will look something like this
+    output = out
+    error = err
+    log = log
 
-    [root@740b9db736a1 /]#
+    queue
 
-You can now install the software that you need through the image package 
-manager, in this case `apt-get`. If you want `/cvmfs` mounted in your image 
-please run
 
-    [root@740b9db736a1 /]# mkdir -p /cvmfs 
+## Frequently Asked Questions / Common Issues
 
-Once you have installed all the software, you simply `exit`
+### I already have a Singularity container, not a Docker one
 
-    [root@740b9db736a1 /]# exit
+?
```

Comment:
This needs to be elaborated on. Unfortunately, I don't know the answer to this myself :)

Reply:
my thoughts exactly. 😂